### PR TITLE
fix extension merge and add name and encoding conflict checking

### PR DESCRIPTION
**Pull request by lhtin**
for issue #115.

### CODE REVIEW

File: parse.py [hunk 1]
```diff
@@ -138,6 +138,17 @@ def process_enc_line(line, ext):
 
     return (name, single_dict)
 
+def same_ext (ext_name, ext_name_list):
+  type1 = ext_name.split("_")[0]
+  for ext_name1 in ext_name_list:
+    type2 = ext_name1.split("_")[0]
+    # "rv" mean insn for rv32 and rv64
```
Comment (ZenithalHourlyRate) [inline parse.py:145]: How about rv128
https://github.com/riscv/riscv-opcodes/blob/master/unratified/rv128_c

Comment (lhtin) [reply parse.py:145]: From the readme's [File Naming Policy](https://github.com/riscv/riscv-opcodes#file-naming-policy), instructions in the file name with the `rv_` prefix support only rv32 and rv64.

Comment (ZenithalHourlyRate) [reply parse.py]: Oh, I misunderstood the logic here. Thanks for pointing out!

**Comment by aswaterman** [comment]
I'm going to leave this to @neelgala to think about and review

**Comment by neelgala** [comment]
@aswaterman can you enable the workflow on this ?

**Comment by pavelkryukov** [comment]
`jol rd rs1 jimm20 6..2=0x1b 1..0=3`.

Here **rs1** and **jimm20** overlap and create ambiguity, I have a disable unit test for that in #119

**Comment by aswaterman** [comment]
done

**Comment by lhtin** [comment]
@neelgala Is there any question please?

**Comment by neelgala** [comment]
@aswaterman this can be merged.

**Review by aswaterman — APPROVED** [review]

**Pull request merged**